buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -498,7 +547,7 @@
       "allowOverride": true
     },
     "AzureContainerSymbolPackageGlob": {
-      "value": "symbols\\*.nupkg",
+      "value": "symbolpkg\\*.nupkg",
```

Comment:
Probably worth removing the override of this variable in `pipelines.json`. Passing this in as a pipebuild var was really only useful for sharing the definition verbatim with CoreFX, which isn't the case anymore.